src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -321,6 +321,11 @@ private bool GetMergedRenameProvider()
         private async Task VerifyMergedOnAutoInsertAsync(VSServerCapabilities mergedCapabilities)
         {
             var triggerCharEnumeration = mergedCapabilities.OnAutoInsertProvider?.TriggerCharacters ?? Enumerable.Empty<string>();
+            var purposefullyRemovedTriggerCharacters = new[]
+            {
+                ">"
```

Comment:
I feel it may be helpful for future readers to add a small comment mentioning that `>` was removed since it's now handled in the RZLS

Reply:
Good idea, i'll actually link to this PR for reference